src/sentry/relay/config/metric_extraction.py

```diff
@@ -169,68 +168,455 @@ def _threshold_to_rules(
-"javascript","navigation",28523,"[405,1036]","[0,0]","[0,0]"
-"python","http.server",12234,"[3,111]","[0,0]","[0,0]"
-"node","http.server",8774,"[2,91]","[0,0]","[0,0]"
-"php","http.server",7260,"[33,212]","[0,0]","[0,0]"
-"python","celery.task",2947,"[19,304]","[0,0]","[0,0]"
-"ruby","rails.request",2923,"[7,105]","[0,0]","[0,0]"
-"javascript","ui.load",2714,"[1708,4302296.25]","[0,0]","[0,0]"
-"cocoa","ui.load",1539,"[94,625]","[0,0]","[0,0]"
-"node","awslambda.handler",1503,"[15,300]","[0,0]","[0,0]"
-"csharp","http.server",1188,"[0,59]","[0,0]","[0,0]"
-"java","http.server",1052,"[3,68.25]","[0,0]","[0,0]"
-"python","serverless.function",1038,"[21,384]","[0,0]","[0,0]"
-"java","ui.load",993,"[70,580]","[0,0]","[0,0]"
-"ruby","active_job",688,"[13,217]","[0,0]","[0,0]"
-"ruby","sidekiq",626,"[12,210]","[0,0]","[0,0]"
-"javascript","default",532,"[182.75,1074.5]","[0,0]","[0,0]"
-"python","asgi.server",370,"[152,840.25]","[0,0]","[0,0]"
-"other","navigation",362,"[1001,3002]","[0,0]","[0,0]"
-"php","console.command",237,"[48,1577]","[0,0]","[0,0]"
-"node","default",215,"[42,302]","[0,0]","[0,0]"
-"node","transaction",211,"[1,50]","[0,0]","[0,0]"
-"go","http.server",192,"[0,41]","[0,0]","[0,0]"
-"ruby","rails.action_cable",186,"[0,6]","[0,0]","[0,0]"
-"python","rq.task",172,"[99,654]","[0,0]","[0,0]"
-"other","pageload",156,"[3000,3000]","[4589.822045672948,4589.822045672948]","[3384.3555060724457,3384.3555060724457]"
-"node","gql",123,"[14,220]","[0,0]","[0,0]"
-"ruby","rack.request",121,"[2,76]","[0,0]","[0,0]"
-"node","test",107,"[14.75,1997]","[0,0]","[0,0]"
-"node","gcp.function.http",103,"[5,426.25]","[0,0]","[0,0]"
-"python","default",91,"[4,462.25]","[0,0]","[0,0]"
-"php","queue.process",88,"[20,319]","[0,0]","[0,0]"
-"python","task",86,"[3,299]","[0,0]","[0,0]"
-"other","http.server",81,"[4,20]","[0,0]","[0,0]"
-"python","websocket.server",74,"[1,124]","[0,0]","[0,0]"
-"php","sentry.test",66,"[0,175.5]","[0,0]","[0,0]"
-"ruby","delayed_job",63,"[6,54]","[0,0]","[0,0]"
-"node","request",60,"[4,239]","[0,0]","[0,0]"
-"python","query",57,"[40,286]","[0,0]","[0,0]"
-"java","navigation",50,"[107,2035]","[0,0]","[0,0]"
-"python","mutation",49,"[8,60]","[0,0]","[0,0]"
-"java","task",49,"[150,727]","[0,0]","[0,0]"
-"other","task",42,"[137,804]","[0,0]","[0,0]"
-"php","http.request",38,"[44,328]","[0,0]","[0,0]"
-"node","execute",38,"[23,215]","[0,0]","[0,0]"
-"node","gcp.function.event",37,"[0,394]","[0,0]","[0,0]"
-"cocoa","ui.action",32,"[716,2668.25]","[0,0]","[0,0]"
-"php","default",31,"[14,74]","[0,0]","[0,0]"
-"cocoa","ui.action.click",29,"[541.75,2988]","[0,0]","[0,0]"
-"node","functions.https.onCall",28,"[12,309]","[0,0]","[0,0]"
-"""
-
-
-def _parse_percentiles(value: str) -> Tuple[float, float]:
-    p25, p75 = map(float, value.strip("[]").split(","))
+#
+# The JSON results down below are however not generated by a one-off query
+# against ClickHouse, but by the S&S team using internal tooling provided by
+# the data team.
```

Comment:
nit: I would remove the one-off query from this comment then.

Reply:
i rephrased it to be less confusing, I want to keep a query somewhere for reference since it's more permanent in github